src/Couchbase.Lite.Shared/API/Database/Database.cs

```diff
@@ -618,8 +617,10 @@ public static void Copy([@NotNull]string path, [@NotNull]string name, [@CanBeNul
                     }
                 }
                 #endif
-
-                return Native.c4db_copyNamed(path, name, &nativeConfig, err);
+                using (var parentDirectory = new C4String(config?.Directory)) {
```

Comment:
new C# is cleaner with the new using syntax (`using var parentDirectory = new C4String(config?.Directory)`)

Reply:
Language version has to be 8+